src/main/java/fr/techad/sonar/GerritConfiguration.java

```diff
@@ -43,7 +44,8 @@
 	private String changeId;
 	private String revisionId;
 
-	public GerritConfiguration(Settings settings) {
+	public GerritConfiguration(Settings settings)
+	{
```

Comment:
Opening bracket on the same line than the declaration. (yeah, we’re missing a coding-style file)